pkg/cmd/release/create/create.go

```diff
@@ -433,13 +452,17 @@ func createRun(opts *CreateOptions) error {
 	return nil
 }
 
-func gitTagInfo(tagName string) (string, error) {
-	cmd, err := git.GitCommand("tag", "--list", tagName, "--format=%(contents:subject)%0a%0a%(contents:body)")
+func gitTagInfo(tagName string) (string, string, error) {
+	cmd, err := git.GitCommand("tag", "--list", tagName, "--format=%(objectname)%0a%(contents:subject)%0a%0a%(contents:body)")
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 	b, err := run.PrepareCmd(cmd).Output()
-	return string(b), err
+	if err != nil || len(b) == 0 {
+		return "", "", err
+	}
+	parts := newlineRE.Split(string(b), 2)
```

Comment:
You don't need a regex to split by newlines. Because of the explicit `--format` string, the only possible newline separator can be `\n`. You can use `strings.SplitN(string(b), "\n", 2)`

Reply:
Good to know, I was unsure how the `git` format flag worked on windows. Will fix that up.